dwds/lib/src/debugging/inspector.dart

```diff
@@ -405,34 +411,13 @@ function($argsString) {
           subclasses: [],
           id: classMetaData.id);
     }
-
-    // Parts are relative paths from the libraryRef uri.
-    var parts =
-        (result.result['result']['value']['parts'] as List).cast<String>();
-    // Note that uris here are scheme based
-    // e.g. org-dartlang-app:///web/main.dart
-    // We will need to normalize the parent if dart-lang/sdk#37336 ever gets
-    // fixed.
-    var parent = libraryRef.uri.substring(0, libraryRef.uri.lastIndexOf('/'));
-    var scriptRefs = [
-      ScriptRef(uri: libraryRef.uri, id: createId()),
-      for (var part in parts)
-        ScriptRef(uri: p.join(parent, part), id: createId())
-    ];
-
-    for (var scriptRef in scriptRefs) {
-      _scriptRefs[scriptRef.id] = scriptRef;
-      _scriptIdToLibraryId[scriptRef.id] = libraryRef.id;
-      _serverPathToScriptRef[DartUri(scriptRef.uri, _root).serverPath] =
-          scriptRef;
-    }
-
+    var scripts = await _getScripts();
```

Comment:
```suggestion
    var scripts = await _scriptRefs;
```

Also consider inlining at the usage site instead of making it a separate variable.

Reply:
Duplicate of Jake's comment, and inlined.